tools/spi_nor_config_generator/flashrom_json_gen.patch

```diff
@@ -0,0 +1,1018 @@
+From bf47bd64c9fd156fa7aa6d5c2ecfadba01fe5b12 Mon Sep 17 00:00:00 2001
```

Comment:
I don't get why this patch exists?

Reply:
There was a script at https://github.com/mithro/litex/blob/spi-module/litex/soc/cores/spi/pull_flashrom_data.py which parsed this file directly.

Reply:
I added pull_flashrom_data.py so the patch is not needed anymore.